lms/djangoapps/teams/tests/test_views.py

```diff
@@ -274,7 +311,7 @@ def test_filter_text_search(self):
     @ddt.data(
         (None, 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
         ('name', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
-        ('open_slots', 200, ['Wind Team', 'Nuclear Team', u'sólar team']),
+        ('open_slots', 200, ['Nuclear Team', u'sólar team', 'Wind Team']),
```

Comment:
Does it mean that all of these teams now have the same number of open slots? We wanted test coverage that the ordering by open slots is working.